src/Form/Model/PasswordChange.php

```diff
@@ -2,13 +2,37 @@
 
 namespace App\Form\Model;
 
-use App\Traits\PlainPasswordTrait;
-use App\Validator\Constraints\PasswordChangeConstraint;
+use App\Validator\Constraints\PasswordPolicy;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Security\Core\Validator\Constraints\UserPassword;
 
-#[PasswordChangeConstraint]
 class PasswordChange
 {
-    use PlainPasswordTrait;
+    #[UserPassword(message: 'form.wrong-password')]
+    private string $password;
 
-    public string $password;
+    #[PasswordPolicy]
+    #[Assert\NotCompromisedPassword(skipOnError: 'true')]
+    #[Assert\NotIdenticalTo(propertyPath: 'password', message: 'form.identical-passwords')]
+    private string $newPassword;
+
+    public function getPassword(): ?string
+    {
+        return $this->password;
+    }
+
+    public function setPassword(?string $password): void
+    {
+        $this->password = $password;
+    }
+
+    public function getNewPassword(): ?string
+    {
+        return $this->newPassword;
+    }
+
+    public function setNewPassword(?string $newPassword): void
```

Comment:
```suggestion
    public function setNewPassword(string $newPassword): void
```

I think we should be unable to `null` the newPassword